Server/Internal/Server/Api.cs

```diff
@@ -0,0 +1,16 @@
+using Grpc.Core;
+using Greet;
+
+namespace Greet.Services;
+
+public partial class GreeterService : MyGreeter.MyGreeterBase

```

Comment:
Where is the GreeterService class instantiated?

Reply:
https://github.com/Azure/mygreeterv3_csharp/blob/a6491c094fb5c056bf16111a19e0526f5f335663/api/v1/Client/Client.cs#L10

Reply:
The GreeterService gets registered in this line:
https://github.com/Azure/mygreeterv3_csharp/blob/a6491c094fb5c056bf16111a19e0526f5f335663/Server/Internal/Server/Server.cs#L22

but it does not get explicitly instantiated in the code. When a gRPC request comes in, the dependency injection (DI) container creates an instance of the GreeterService class to handle the request.